projects/plugins/protect/src/class-jetpack-protect.php

```diff
@@ -63,6 +63,10 @@ function () {
 		);
 
 		My_Jetpack_Initializer::init();
+
+		if ( Status::has_vulnerabilities() ) {
+			add_action( 'admin_notices', array( $this, 'jetpack_protect_detect_vuls_notice' ) );
```

Comment:
This class is initialized on the root of the main plugin file, and outside any hook. Calling `has_vulnerabilities` from there is unsafe, as it might need to make a request to the server and it may be too soon. 

In fact I'm getting a Fatal error on my env.

We could hook it to init, admin_init, plugins_loaded or something.